src/js/view/utils/clickhandler.js

```diff
@@ -0,0 +1,72 @@
+define([
+    'utils/ui',
+    'events/events',
+    'utils/backbone.events',
+    'utils/underscore'
+], function(UI, events, Events, _) {
```

Comment:
I think it would be better to replace the clickHandlers when needed instead of doing this `alternateClickHandler` stuff.